src/interpreter/ByteCodeInterpreter.cpp

```diff
@@ -1343,6 +1343,39 @@ Value ByteCodeInterpreter::interpret(ExecutionState& state, ByteCodeBlock* byteC
                 return result;
             }
 
+            DEFINE_OPCODE(BlockOperation)
+                :
+            {
+                BlockOperation* code = (BlockOperation*)programCounter;
+                size_t newPc = programCounter;
+                Value* stackStorage = registerFile + byteCodeBlock->m_requiredRegisterFileSizeInValueSize;
+                Value v = blockOperation(state, code, newPc, byteCodeBlock, registerFile, stackStorage);
+                if (!v.isEmpty()) {
+                    return v;
+                }
+                if (programCounter == newPc) {
+                    return Value();
+                }
+                programCounter = newPc;
+                NEXT_INSTRUCTION();
+            }
+
+            DEFINE_OPCODE(SetConstBinding)
+                :
+            {
+                SetConstBinding* code = (SetConstBinding*)programCounter;
+                state.lexicalEnvironment()->record()->asDeclarativeEnvironmentRecord()->asDeclarativeEnvironmentRecordNotIndexed()->setAsImmutableBinding(state, code->m_name);
+                ADD_PROGRAM_COUNTER(SetConstBinding);
+                NEXT_INSTRUCTION();
+            }
+
+            DEFINE_OPCODE(NoOperation)
```

Comment:
Could we optimize this out at some point?

Reply:
Sure, I updated the PR with removing it.